src/pages/home/index.js

```diff
@@ -51,7 +51,7 @@ const Home = () => {
   const parseTargets = () => {
     let targetsParsedList = [];
     targets?.targets.map(target => {
-      targetsParsedList.push([target.target.lat, target.target.lng, target.target.title]);
+      targetsParsedList.push([target.target.lat, target.target.lng, target.target.id]);
```

Comment:
Ohhh I think the above coment of Array vs object is originated here 😄 What do you think about treating this as an object instead of an Array? I think the nature of what you're pushing into the array fits better an object that an Array structure as they're attributes that represent a `Target` entity.

Reply:
Done!